js/scenes/Events/EventsContainer.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { ActivityIndicator } from 'react-native';
+import { connect } from 'react-redux';
+
+import Events from './Events'
+
+class EventsContainer extends Component {
+    static route = {
+        navigationBar: {
+          title: 'Events',
+        }
+      }
+
+      componentDidMount() {
+          // ADD DISPATCH FUNCTION HERE TO FETCH DATA
+    }
+
+
+
+    render() {
+        // if (this.props.loading) {
+        //     return (
```

Comment:
Please don't commit commented out code with your pull request. Remove this commented out code.